heft-plugins/heft-sass-plugin/src/SassTypingsPlugin.ts

```diff
@@ -51,7 +58,11 @@ export class SassTypingsPlugin implements IHeftPlugin {
 
     await sassTypingsGenerator.generateTypingsAsync();
     if (isWatchMode) {
-      Async.runWatcherWithErrorHandling(async () => await sassTypingsGenerator.runWatcherAsync(), logger);
```

Comment:
This method was added by @iclanton to cleanup floating promises. Do you think it's safe to go back to a simple try-catch?

Reply:
`try { await foo() } catch {}` would be safe, however it is a change in behavior. I missed that `runWatcherWithErrorHandling()` is fire and forget and my code would wait on the promise to complete. So I've make a copy of `Async.runWatcherWithErrorHandling()` and reverted to the same code. Thanks for pointing it out!